Platform/ApollolakeBoardPkg/BoardConfig.py

```diff
@@ -95,7 +95,14 @@ def __init__(self, *args, **kwargs):
 			self.STAGE1B_SIZE += 0x2000
 		self.STAGE2_SIZE          = 0x00032000
 		self.PAYLOAD_SIZE         = 0x0001F000
-		self.EPAYLOAD_SIZE        = 0x00120000
+
+		if len(self._PAYLOAD_NAME.split(';')) > 1:
+			# EPAYLOAD is specified
+			self.EPAYLOAD_SIZE      = 0x00120000
+		else:
+			# EPAYLOAD does not exist, create a dummy one
+			self.EPAYLOAD_SIZE      = 0x1000
```

Comment:
If dummy EPAYLOAD is required for stitching tool, why don't you remove 'if self.EPAYLOAD_SIZE > 0:' in GetImageLayout()?

Reply:
good point.  let me remove that.

Reply:
updated the patch to remove it.